SLRemoting/SLObject.m

```diff
@@ -229,7 +229,7 @@ + (NSDictionary *)convertArguments:(NSDictionary *)prop {
         id value = [prop objectForKey:key];
         if ([value isKindOfClass:[NSDate class]]) {
             NSString *jsonString = [value convertToJSONString];
-            value = @{ @"$type": @"date", @"$data": jsonString };
+            value = jsonString ;
```

Comment:
@bajtos :
Update: 
` value = @{ @"$type": @"date", @"$data": jsonString };`

gets converted to following URL string, which is not coerced correctly 
`http://localhost:3000/widgets/testDate?date[%24data]=1970-01-01T00%3A02%3A03.000Z&date[%24type]=date`